app/src/main/java/net/gaast/giggity/Schedule.java

```diff
@@ -298,7 +303,11 @@ public void loadSchedule(String url_, Fetcher.Source source) throws IOException
 				loadDeox(in);
 			} else if (head.contains("begin:vcalendar")) {
 				loadIcal(in);
-			} else {
+			}
+			  else if(head.contains("{")&&head.contains("[")){
```

Comment:
On top of this in theory this could rely on MIME types (mostly not possible elsewhere because it's all XML). But not terribly important I guess. I dislike how fuzzy this is but I've never really been able to do better anyway.